src/services/sheltersService.js

```diff
@@ -29,6 +30,35 @@ const SheltersService = {
       (!res.ok)
         ? res.json().then(e => Promise.reject(e))
         : res.json());
+  },
+  updateShelter(shelter) {
+    console.log("postShelter")
+    console.log(shelter.shelterID + ".")
+    console.log(shelter)
+    return fetch(`${HOSTNAME}/shelters/${shelter.shelterID}`, {
+      method: 'PATCH',
+      headers: {
+        'content-type': 'application/json',
+        'authorization': AuthService.getToken()
+      },
+      body: JSON.stringify({ ...shelter })
```

Comment:
This is a question for my own edification. Why use `{ ...shelter}` instead of `shelter`? I still have yet to fully grasp the usage of `...` in JS

Reply:
In this case, ... formats an array to json nicely (I believe but I might be wrong!), but I am simply passing a json object so I removed it!